src/System.Composition.Convention/src/System/Composition/Convention/ConventionBuilder.cs

```diff
@@ -0,0 +1,351 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Reflection;
+
+using Microsoft.Internal;
+
+namespace System.Composition.Convention
+{
+    /// <summary>
+    /// Entry point for defining rules that configure plain-old-CLR-objects as MEF parts.
+    /// </summary>
+    public class ConventionBuilder : AttributedModelProvider
+    {
```

Comment:
@KrzysztofCwalina I don't really follow.  They are already the only public types exposed by System.Composition.Convention, as far as I can see.